testdata/cases/server64/server64.conf

```diff
@@ -0,0 +1,67 @@
+
+service {
+	user root
+	group root
+	pidfile /dummy/file/path1
+	proto-fd-max 15000
+
+    secrets-address-port test_dns_name 4000 127.0.0.1
+    secrets-tls-context tlscontext
+    secrets-uds-path /test/path/socket
+}
+
+logging {
+	
+	file /dummy/file/path2 {
+		context any info
+	}
+}
+
+network {
+	service {
+		address any
+		port 3000
+	}
+
+	heartbeat {
+		mode multicast
+		multicast-group 127.0.0.1
+		port 9918
+
+		
+		
+
+		interval 150
+		timeout 10
+	}
+
+	fabric {
+		port 3001
+	}
+
+	info {
+		port 3003
+	}
+}
+
+namespace ns1 {
+	replication-factor 2
+	memory-size 4G
+
+	index-type flash {
+        mount /dummy/mount/point1 /test/mount2
+        mounts-high-water-pct 30
+        mounts-size-limit 10G
+    }
```

Comment:
Might be good to test `sindex-type flash` too since that is new in 6.4

Reply:
Good suggestion, sindex-type wasn't being parsed, I've added it to the management lib and this test.